### DISCUSSION TITLE
TNT INTEGRATION

### DISCUSSION THREAD

**Discussion by taragnolini** [karrioapi/karrio#524] [General]
Hello,
i am trying to use the TNT carrier. 
I filled it with my account code and password, but when i try for example to track a parcel or create a shipment i always got a 403 "Unauthorized response" - I see in TNT documentation, that i should pass the Basic Authentication, shall i include it in the meta of the carrier configuration?
thanks

**Comment by danh91** [comment]
Hi @taragnolini ,
I hope you were able to chat with @rcknr.
Keep me posted in your email exchanges where I can help

**Comment by danh91** [comment]
Notes: For anyone stumbling upon this or having similar issues,
TNT in Italy requires the v2 version of their API whereas Karrio currently has v3 integrated.
I imagine that adding the right integration will help solve the authentication issue.
